lib/arena.rb

```diff
@@ -1,2 +1,90 @@
+# require 'pry'
+require_relative 'gladiator'
+
 class Arena
+
+  attr_accessor :name, :gladiators, :battle, :audience_response
+
+  def initialize(name)
+    @name = name.capitalize
+    @gladiators = []
+    @battle = []
+    @gladiator_names = []
+    @audience_response = ""
+  end
+
+  def add_gladiator(gladiator)
+    if @gladiators.length < 2
+      @gladiators << gladiator
+    end
+  end
+
+  def fight
+    if @gladiators.length == 2
+      @battle = @gladiators.map do |gladiator|
+        gladiator.weapon
+      end
+
```

Comment:
Good job on this fight method, however could you think of a way to shorten this into fewer lines of code?